assets/airports/lipz.json

```diff
@@ -0,0 +1,2300 @@
+                ["aar/long", 1],
+                ["afl/b738", 6],
+                ["asl", 2],
+                ["aua", 12],
+                ["ely", 2],
+                ["ezy", 9],
+                ["fin", 2],
+                ["lot", 5],
+                ["pgt", 2],
+                ["qtr/long", 3],
+                ["svr", 1],
+                ["tar", 1],
+                ["thy", 9],
+                ["uae", 3],
+                ["voe", 14]
+            ]
+     
+        },
+        {
+            "origin": "LIPZ",
+            "destination": "",
+            "category": "departure",
+            "route": "LIPZ04R.VIC5L.VIC.L615.ADOSA",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 3.2,
+            "airlines": [
+                ["baw", 20],
+                ["dal/long", 3],
+                ["ezy", 16],
+                ["tra", 1],
+                ["ual/long", 3],
+                ["voe", 2]
+            ]
+        },
+        {
+            "origin": "LIPZ",
+            "destination": "",
+            "category": "departure",
+            "route": "LIPZ04R.ROKIB5L.ROKIB",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 7.4,
+            "airlines": [
+                ["aal/long", 3],
+                ["bel", 5],
+                ["bti", 2],
+                ["dla", 13],
+                ["dlh", 17],
+                ["ewg", 16],
+                ["ezy", 18],
+                ["fin", 2],
+                ["klm", 11],
+                ["lgl", 2],
+                ["nax", 2],
+                ["ryr", 12],
+                ["swr", 9],
+                ["sxs", 2]
+            ]
+        },
+        {
+            "origin": "LIPZ",
+            "destination": "",
+            "category": "departure",
+            "route": "LIPZ04R.ROSKA5L.ROSKA.T132.ROTAR",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 5.4,
+            "airlines": [
+                ["asl", 2],
+                ["ely", 2],
+                ["ezy", 14],
+                ["pgt", 2],
+                ["qtr/long", 3],
+                ["uae", 3],
+                ["voe", 18],
+                ["aea", 7],
+                ["ibe", 11],
+                ["tap", 5],
+                ["thy", 9],
+                ["vlg", 10]
+            ]
+        },
+        {
+            "origin": "LIPZ",
+            "destination": "",
+            "category": "departure",
+            "route": "LIPZ04R.CHI5L.CHI.L612.BELOV",
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 3.4,
+            "airlines": [
+                ["aea", 7],
+                ["ezy", 2],
+                ["ibe", 11],
+                ["tap", 5],
+                ["thy", 9],
+                ["vlg", 10],
+                ["voe", 3]
+            ]
+        }
+    ],
+    "maps": [
```

Comment:
Video map looks pretty good! There are a couple of things that are bugging me though.
1. The VOR symbols don't quite sit on the VORs:
![image](https://user-images.githubusercontent.com/13125311/74380554-b94c7f00-4de1-11ea-806c-b1d08520a1af.png)
2. The airspace, video map and restricted areas are all slightly off. (This may be true in real life, but it's weird)
![image](https://user-images.githubusercontent.com/13125311/74380676-eb5de100-4de1-11ea-81de-ecf7db9fd1ca.png)
3. Looks like there are some lines missing to the West
![image](https://user-images.githubusercontent.com/13125311/74380745-0e889080-4de2-11ea-9525-c8ed6905d426.png)
4. The extended centerline is very inconsistent. I'd strongly suggest using the qgsopenscope plugin for qgis to generate it automatically
![image](https://user-images.githubusercontent.com/13125311/74380819-37108a80-4de2-11ea-836e-0db5c916ca25.png)

Reply:
![RadarVideoMapReal](https://user-images.githubusercontent.com/43710005/74544709-e2871f80-4f47-11ea-99e3-53f5eea2db10.jpeg)

This is the real video map, I tried to make it as best as I could.
1.  i'll fix the navaid positions.
2. as far as the restricted, I can mach the 2 side lines to make it prettier. below theres a snip from skyvector.
![image](https://user-images.githubusercontent.com/43710005/74545242-cf288400-4f48-11ea-9459-0a0a98aa1f43.png)
3. just corn fields
4. what do you mean by inconsistent? It's exactly like the real one.

Reply:
1. roger
2. I wouldn't trust SkyVector for restricted airspace coordinates, it can be incredibly inaccurate. I would suggest using the coordinates given in the AIP. It might still be 'off' there but at least we'll know that's what it's meant to look like.
3. I wasn't clear, my bad. I meant that the video map lines going off to the west just stop suddenly. I assume they connect to some other line(s) in the real scope (I can't tell by looking at the real scope picture either).
4. Disregard then 😄 It's a very weird looking extended centerline but it's realistic so it can stay.

Reply:
2. yes, skyvector was just an example. It's supposed to be this way.
This is AIP
![image](https://user-images.githubusercontent.com/43710005/74592732-a07fdb80-5024-11ea-85e2-b4174ec5068a.png)
3. I suppose also, but couldn't get better info. The only thing I have is that picture + AIP.
4. artistic ;)